dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/objectbundle/hooks/JobConfigurationObjectBundleHook.java

```diff
@@ -173,7 +171,13 @@ public void setSchedulingManager( SchedulingManager schedulingManager )
         ErrorReport jobValidation = job.validate();
         if ( jobValidation != null )
         {
-            errorReports.add( jobValidation );
+            // If the error is caused by the environment and the job is a non configurable job
+            // that exists already, then the error can be ignored. Job has the issue with and
+            // without updating it.
+            if ( (jobValidation.getErrorCode() != ErrorCode.E7010) || (persitedJobConfiguration == null) || jobConfiguration.isConfigurable() )
```

Comment:
You dont need the extra paranthesis inside the if statement here. If you do use them, remember spaces between them and their content.